metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1801,7 +1801,7 @@ class MetalsLanguageServer(
         item <- i.sources.getItems.asScala
         source <- item.getSources.asScala
       } {
-        val sourceItemPath = source.getUri.toAbsolutePath
+        val sourceItemPath = source.getUri.toAbsolutePath(followSymlink = false)
```

Comment:
This is needed so that we are able to be able to figure out what the original source roots were.